app/api/v1/endpoints/strategies.py

```
@@ -1114,45 +1205,70 @@ async def submit_strategy_for_review(
+            )
+
+            db.add(submission)
+            await db.commit()
+            await db.refresh(submission)
+
+            submission_id = submission.id
+            persisted = True
+
+        except (sa_exc.NoSuchTableError, sa_exc.OperationalError, sa_exc.ProgrammingError) as db_error:
+            # If database table doesn't exist, generate a temporary ID
+            logger.warning(f"Database table issue, using temporary ID: {str(db_error)}")
```

Ruff (0.12.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)